packages/labs/react/src/test/create-component_test.tsx

```diff
@@ -333,6 +340,19 @@ suite('createComponent', () => {
     assert.equal(el.firstElementChild!.localName, 'div');
   });
 
+  test('can set multiple children to multiple slots', async () => {
+    const children = window.React.createElement('div');
+    const foos = window.React.createElement('div');
```

Comment:
We need to test that you can specify a a set of children for a slot. Ideally we'd be testing with JSX here as well. I think in that case it would be a JSX fragment and without JSX I guess you can make that via a `React.createElement(React.Fragment, ...`

Reply:
Nice! Yah I'll add that test case